packages/quantic/force-app/main/default/lwc/quanticSearchInterface/quanticSearchInterface.js

```diff
@@ -168,6 +178,27 @@ export default class QuanticSearchInterface extends LightningElement {
     );
   }
 
+  bindAriaLiveEvents() {
+    this.template.addEventListener('arialivemessage', this.handleAriaLiveMessage.bind(this));
+    this.template.addEventListener('registerregion', this.handleRegisterAriaLiveRegion.bind(this));
+  }
+
+  handleAriaLiveMessage(event) {
+    /** @type {Object} */
```

Comment:
If possible, I would use the `QuanticAriaLive` type here.

Reply:
The problem is that selecting that Element with `querySelector` doesn't give me an "instance" of QuanticAriaLive as far as the types are concerned. I've looked for a way to fix this without success :(

I created an interface instead:

```
/**
* @typedef {Object} IQuanticAriaLive
* @method registerRegion
* @method updateMessage
*/
```

That was the only way I found to have it working